### DISCUSSION TITLE
Write TB calendar into Google Calendar ?

### DISCUSSION THREAD

**Discussion by nitwoo** [kewisch/gdata-provider#539] [Support]
Hi there,

Sorry if this is a dumb question, I've looked at the faq / home page but didn't see it anywhere.
Basically, after adding a Google Calendar via the provider, is there a way to synchronize / always "auto" add an event to both calendars, both the TB and Google one ?

I saw the import feature in the event and tasks panel, but I was wondering if that can be me automated and put on default settings somehow ?

Thanks.

**Comment by xjonx1** [comment]
There was an issue with the add-on and is now fixed by the developer. Details here:https://github.com/kewisch/gdata-provider/discussions/519#discussioncomment-4147395
